### ISSUE TITLE
Images and elements with background images should have descriptive screen reader text

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1059]
This does not include icons or other visual non-text elements.  This only ensures Image tags have valid and descriptive alt attributes and that background images for non-decorative elements have associated descriptive text
